assets/js/modules/analytics-4/components/widgets/TopConvertingTrafficSourceWidget.js

```diff
@@ -37,14 +49,92 @@ export default function TopConvertingTrafficSourceWidget( {
 		select( CORE_USER ).isKeyMetricsWidgetHidden()
```

Comment:
This must have been scaffolded before, but there is a utility used by other widget tiles to simplify this which also prevents side effects from the widget rendering without the need to have conditional selects https://github.com/google/site-kit-wp/blob/96cf06147133fa263adff5e50363f11d5b574f4a/assets/js/modules/analytics-4/components/widgets/LoyalVisitorsWidget.js#L128

It might not be worth changing at this point as this should be covered by #7061

Reply:
Thanks @aaemnnosttv, that's useful to know. It does seem this should conceptually be covered by #7061, but looking at the IB, it only specifies the removal of the `whenKeyMetricsWidgetVisible` HOC, with the only reference to `keyMetricsWidgetHidden` being in a struck-out line.

Noting that there are other widgets currently doing a similar check with `isKeyMetricsWidgetHidden()`, and furthermore that #7061 is currently in execution with @jimmymadon, I'd say it's probably _not_ worth updating this widget, but rather getting it merged while Jimmy's still working on #7061, and dropping him a line to make sure he merges `develop` into his branch and updates all of these widgets to remove the checks based on `isKeyMetricsWidgetHidden()`.

```sh
$ grep -l isKeyMetricsWidgetHidden assets/js/**/*Widget.js

assets/js/modules/analytics-4/components/widgets/EngagedTrafficSourceWidget.js
assets/js/modules/analytics-4/components/widgets/PopularProductsWidget.js
assets/js/modules/analytics-4/components/widgets/TopCitiesWidget.js
assets/js/modules/analytics-4/components/widgets/TopConvertingTrafficSourceWidget.js
assets/js/modules/analytics-4/components/widgets/TopCountriesWidget.js
assets/js/modules/analytics-4/components/widgets/TopTrafficSourceWidget.js
```

Reply:
Update: Having spoken to Jimmy on Slack he's happy to take the approach of merging `develop` into his branch to update this component along with the others.